ci-operator/step-registry/ipi/conf/azurestack/ipi-conf-azurestack-commands.sh

```diff
@@ -42,5 +54,7 @@ compute:
   replicas: ${workers}
 EOF
 
+cat "${CONFIG}"
+
```

Comment:
Is this added temporarily for debugging purposes?

Reply:
yes, removed.